docs/changehistory/NextVersion.md

```diff
@@ -27,6 +27,10 @@ Added a new helper type [ListenerType]($core-bentley) to retrieve the listener t
 
 The Xml, JSON and interfaces all have an `appliesTo` property of type `CustomAttributeContainerType`, only the implementation had a `containerType` property. The existing `containerType` getter was marked as deprecated and `appliesTo` getter was added with no behavioral changes. The protected `_containerType` variable was renamed to `_appliesTo` for consistency even though this is a breaking change for any class that extends `CustomAttributeClass`, no know derived classes exist outside of the core packages.
 
+## ecschema-rpcinterface-common methods GET by default
+
+Made all ECSchemaRpcInterface methods GET by default so responses can be cached w/o each app setting flags on their own
```

Comment:
ECSchemaRpcInterface is `@internal`. Can you rephrase to explain how this surfaces to users?

Reply:
@ColinKerr should it even be internal? AFAIK, it's not automatically registered by `IModelApp`/`IModelHost` like some others are, so I suppose it's expected to be directly used by consumers, no?

Reply:
The RPC release tags are currently a mess. All the RPC interfaces are internal, yet apps have to be able to refer to them in order to register them. The rest of the subsystem is "deprecated" with no replacement, and much of it is also internal. @aruniverse and @paulius-valiunas are apparently talking to @swbsi about potential changes in 5.0.

I will edit Colin's comments for legibility and ignore the @internal thing.

Reply:
I was wrong - some of the RPC **implementations** are registered by `IModelHost` automatically. On the frontend, the interfaces need to be passed to RPC initialization function by consumers (see [here](https://github.com/iTwin/itwinjs-core/blob/aadb27c508e517ebed56074638bbe562a680b034/test-apps/display-test-app/src/frontend/App.ts#L302)). So all these interfaces should be public, and most of them aren't at the moment (examples: [IModelReadRpcInterface](https://github.com/iTwin/itwinjs-core/blob/aadb27c508e517ebed56074638bbe562a680b034/core/common/src/rpc/IModelReadRpcInterface.ts#L74), [SnapshotIModelRpcInterface](https://github.com/iTwin/itwinjs-core/blob/aadb27c508e517ebed56074638bbe562a680b034/core/common/src/rpc/SnapshotIModelRpcInterface.ts#L20)). IMO they all, including `ECSchemaRpcInterface` should be public.

Reply:
Yes, by our policies the interfaces should be public and all of their guts should be internal. Waiting to discuss with @aruniverse et al before making any changes.

I rewrote NextVersion.md. @ColinKerr please resolve comment if satisfied.

Reply:
I rewrote the update to focus on the public API that is impacted, the ECSchemaRpcLocater.  I'll let someone else resolve now that I've had the last word.